assets/xml/objects/object_po_fusen.xml

```diff
@@ -1,32 +1,40 @@
-        <Limb Name="object_po_fusen_Standardlimb_002478" Type="Standard" EnumName="OBJECT_PO_FUSEN_LIMB_03" Offset="0x2478" />
-        <Limb Name="object_po_fusen_Standardlimb_002484" Type="Standard" EnumName="OBJECT_PO_FUSEN_LIMB_04" Offset="0x2484" />
-        <Limb Name="object_po_fusen_Standardlimb_002490" Type="Standard" EnumName="OBJECT_PO_FUSEN_LIMB_05" Offset="0x2490" />
-        <Limb Name="object_po_fusen_Standardlimb_00249C" Type="Standard" EnumName="OBJECT_PO_FUSEN_LIMB_06" Offset="0x249C" />
-        <Limb Name="object_po_fusen_Standardlimb_0024A8" Type="Standard" EnumName="OBJECT_PO_FUSEN_LIMB_07" Offset="0x24A8" />
-        <Limb Name="object_po_fusen_Standardlimb_0024B4" Type="Standard" EnumName="OBJECT_PO_FUSEN_LIMB_08" Offset="0x24B4" />
-        <Limb Name="object_po_fusen_Standardlimb_0024C0" Type="Standard" EnumName="OBJECT_PO_FUSEN_LIMB_09" Offset="0x24C0" />
-        <Skeleton Name="object_po_fusen_Skel_0024F0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_PO_FUSEN_LIMB_NONE" LimbMax="OBJECT_PO_FUSEN_LIMB_MAX" EnumName="ObjectPoFusenLimb" Offset="0x24F0" />
+        <!-- Empty, contains the data "0x0000, 0xC000". -->
+        <!-- PoFusen is animated directly from its code. -->
+        <Animation Name="gPoeBalloonEmptyAnim" Offset="0x40" />
+
+        <DList Name="gPoeBalloonChainAndLanternDL" Offset="0x8C0" />
+        <DList Name="gPoeBalloonLeftHandDL" Offset="0xAF8" />
+        <DList Name="gPoeBalloonLeftForarmDL" Offset="0xB90" />
+        <DList Name="gPoeBalloonLeftUpperarmDL" Offset="0xC48" />
+        <DList Name="gPoeBalloonRightHandDL" Offset="0xCD8" />
+        <DList Name="gPoeBalloonRightForarmDL" Offset="0xD70" />
+        <DList Name="gPoeBalloonRightUpperarmDL" Offset="0xE28" />
+        <DList Name="gPoeBalloonBodyDL" Offset="0xEB8" />
+
+        <Texture Name="gPoeBalloonTLUT" OutName="balloon_tlut" Format="rgba16" Width="16" Height="16" Offset="0x10E0" />
+        <!-- The tattered cloth along the eyes are part of the face texture, but not below the chin. -->
+        <Texture Name="gPoeBalloonTatteredChinTex" OutName="tattered_chin" Format="rgba16" Width="16" Height="16" Offset="0x12E0" />
+        <Texture Name="gPoeBalloonLanternTopTex" OutName="lantern_top" Format="rgba16" Width="16" Height="16" Offset="0x14E0" />
+        <Texture Name="gPoeBalloonLanternStandTex" OutName="lantern_stand" Format="rgba16" Width="16" Height="16" Offset="0x16E0" />
+        <Texture Name="gPoeBalloonLanternGlassTex" OutName="lantern_glass" Format="ci8" Width="16" Height="16" Offset="0x18E0" />
+        <Texture Name="gPoeBalloonHandTex" OutName="the_hand" Format="i4" Width="16" Height="16" Offset="0x19E0" />
+        <!-- Think this is just extra texture for the orange skin surface. -->
+        <Texture Name="gPoeBalloonOrangeBlotchTex" OutName="orange_blotch" Format="rgba16" Width="16" Height="16" Offset="0x1A60" />
```

Comment:
I edited the texture and rebuilt the game, and it's pretty clear what this is. It's basically all the orange that isn't part of the "face". I don't quite know what to name it, but hopefully you can come up with something:
![image](https://user-images.githubusercontent.com/12245827/211120186-ad07bc73-077c-4f0f-ba75-86103c85378c.png)

Reply:
so just the whole of the skin texture, I would probably call it orange skin?

Reply:
Hmm, it's a bit tricky because you have to ask "does a balloon really have skin?". But balloons don't typically have faces either but this one does. I'd ultimately be okay with "skin", though I'm wondering if there's a better term.